lib/manageiq/rpm_build/generate_core.rb

```diff
@@ -97,6 +99,34 @@ def plugin_paths
         end
       end
 
+      def fixup_sti_loader!
+        sti_loader_yml_path = miq_dir.join("tmp/cache/sti_loader.yml")
+
+        sti_loader = YAML.load_file(sti_loader_yml_path)
+        sti_loader.transform_keys! do |path|
+          if !path.start_with?(BUILD_DIR.to_s)
+            path
+          else
+            rel_path = Pathname.new(path).relative_path_from(BUILD_DIR)
+            prefix, target_path = rel_path.to_s.split("/", 2)
+
+            prefix =
+              case prefix
+              when OPTIONS.product_name
+                "/var/www/miq/vmdb"
+              when /^#{OPTIONS.product_name}-gemset/
+                File.join("", "opt", OPTIONS.rpm.org_name, "#{OPTIONS.product_name}-gemset")
```

Comment:
For some reason, I'm more comfortable with 

```suggestion
                File.join("", "opt", OPTIONS.rpm.org_name, prefix)
```

I guess I don't like our use of `product_name` everywhere in this code

Reply:
The prefix actually includes the version number as well (which is why a regex is used in the case statement), so we have to rebuild the value.

Reply:
That being said, I could see possibly DRYing up those two calls with something like 

```ruby
gemset_prefix = "#{OPTIONS.product_name}-gemset"
...
case prefix
when /^#{gemset_prefix}/
  File.join("", "opt", OPTIONS.rpm.org_name, gemset_prefix)
```

Reply:
:+1: done

Reply:
didn't see the regex part. thnx.

yea, leave it as is.